src/AirtableClient.php

```diff
@@ -39,29 +39,18 @@ class AirtableClient
      */
     public function __construct(string $baseId, Table $table)
     {
-        $this->client = new Client([
```

Comment:
I like the way you're thinking. But sorry, I don't agree with this move to consolidate the two methods to just a construct. There will be more methods in this object later, I think. (such as get(), put(), post(), delete()) The getClient() method should remain.

At some future point we will likely also refactor the construct to take a factory, removing the need to instantiate the client.

Reply:
I don't think I changed this any much. This is latest commit https://github.com/Beachcasts/airtable-sdk-php/blob/24d96298dad0864edd5d3a14785fb590093ac26b/src/AirtableClient.php file.